hail/src/main/scala/is/hail/expr/ir/functions/LocusFunctions.scala

```diff
@@ -422,7 +422,7 @@ object LocusFunctions extends RegistryFunctions {
             val srcRG = iT.pointType.asInstanceOf[PLocus].rg
             val destRG = rt.types(0).asInstanceOf[PInterval].pointType.asInstanceOf[PLocus].rg
             val er = EmitRegion(cb.emb, r)
-            val intervalObj = Code.checkcast[Interval](asm4s.coerce[AnyRef](wrapArg(er, iT)(interval.tcode[Long])))
+            val intervalObj = Code.checkcast[Interval](asm4s.coerce[AnyRef](wrapArg(er, iT)(interval.asInstanceOf[SIntervalPointerCode].a)))
```

Comment:
There's a `scodeToJavaValue` method in Functions.scala -- use that instead. This is wrong for the same reason as above.